util/chunk/row.go

```diff
@@ -221,35 +221,35 @@ func (r Row) ToString(ft []*types.FieldType) string {
 	var buf []byte
 	for colIdx := 0; colIdx < r.Chunk().NumCols(); colIdx++ {
 		if r.IsNull(colIdx) {
-			buf = append(buf, "nil, "...)
-			continue
-		}
-		switch ft[colIdx].EvalType() {
-		case types.ETInt:
-			buf = strconv.AppendInt(buf, r.GetInt64(colIdx), 10)
-		case types.ETString:
-			switch ft[colIdx].Tp {
-			case mysql.TypeEnum:
-				buf = append(buf, r.GetEnum(colIdx).String()...)
-			case mysql.TypeSet:
-				buf = append(buf, r.GetSet(colIdx).String()...)
-			default:
-				buf = append(buf, r.GetString(colIdx)...)
-			}
-		case types.ETDatetime, types.ETTimestamp:
-			buf = append(buf, r.GetTime(colIdx).String()...)
-		case types.ETDecimal:
-			buf = append(buf, r.GetMyDecimal(colIdx).ToString()...)
-		case types.ETDuration:
-			buf = append(buf, r.GetDuration(colIdx, ft[colIdx].Decimal).String()...)
-		case types.ETJson:
-			buf = append(buf, r.GetJSON(colIdx).String()...)
-		case types.ETReal:
-			switch ft[colIdx].Tp {
-			case mysql.TypeFloat:
-				buf = strconv.AppendFloat(buf, float64(r.GetFloat32(colIdx)), 'f', -1, 32)
-			case mysql.TypeDouble:
-				buf = strconv.AppendFloat(buf, r.GetFloat64(colIdx), 'f', -1, 64)
+			buf = append(buf, "NULL"...)
+		} else {
+			switch ft[colIdx].EvalType() {
```

Comment:
Why replace `continue` with `else`?

Reply:
If `colIdx` is the last column of this row, we shouldn't put the comma(`, `) after `nil` nor `NULL`.